### PR TITLE
Added graphql client modules

### PR DESCRIPTION
**Pull request by rpanic**

### CODE REVIEW

File: packages/sdk/test/graphql.test.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    // const server = await startServer();
+    //
+    // await sleep(2000);
+
+    appChain = prepare();
+
+    await appChain.start();
+  })
+
+  it.skip("should retrieve state", async () => {
```
Comment (maht0rz) [inline packages/sdk/test/graphql.test.ts]: I'd like to see a test where we submit a tx, produce a block and then read the state, checking the tx has been successfull too

File: packages/sdk/test/graphql.test.ts [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+  return appChain
+}
+
+describe("graphql client test", function() {
+
+  let appChain: ReturnType<typeof prepare> | undefined = undefined
+
+  beforeAll(async () => {
+    // const server = await startServer();
```
Comment (maht0rz) [inline packages/sdk/test/graphql.test.ts]: looks like this isnt used, so the helper fn for sleep can be removed too, or moved to common utils

File: packages/sdk/src/graphql.ts
Comment (maht0rz) [inline packages/sdk/src/graphql.ts:1]: probably should be removed in favour of test/graphql.test.ts

File: packages/sdk/test/graphql.test.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  }).sign(pk);
+
+  console.log(tx.toJSON());
+
+  return tx;
+}
+
+function prepare(){
+
+  const appChain = AppChain.from({
```
Comment (maht0rz) [inline packages/sdk/test/graphql.test.ts]: would be great if we had to separate AppChains running, one for the server side and one for the client side, then this test would be perfect

**Review by maht0rz — CHANGES_REQUESTED** [review]

File: packages/sdk/test/graphql.test.ts (hunk 3 above)
Comment (rpanic) [reply packages/sdk/test/graphql.test.ts]: we do

**Pull request merged**